lib/tic_tac_toe/position_validator.ex

```diff
@@ -1,12 +1,14 @@
 defmodule TicTacToe.PositionValidator do
-  alias TicTacToe.Board
+  alias TicTacToe.BoardInspect
 
   def error(position, board) do
+    board = BoardInspect.get(board, position)
+
     cond do
       Enum.member?(1..9, position) == false ->
         "Invalid entry."
 
-      Board.available_positions(board) |> Enum.member?(position) == false ->
+      board.position_value ->
```

Comment:
I mentioned this in our sync, but be cautious of introducing dependencies on the underlying data structure. How does that affect our ability to change the behavior by injecting a different data structure?

No change needed, just something to consider.

Reply:
perhaps get does not need to return a data structure?